src/ui/server/helpers/summary-lists/eligibility/eligibility-fields/index.ts

```diff
@@ -54,14 +51,6 @@ const generateEligibilityFields = (answers: InsuranceEligibility) => {
       },
       FIELDS_ELIGIBILITY[HAS_MINIMUM_UK_GOODS_OR_SERVICES].ANSWER,
     ),
-    fieldGroupItem(
-      {
-        field: getFieldById(FIELDS_ELIGIBILITY, WANT_COVER_OVER_MAX_AMOUNT),
-        data: answers,
-        renderChangeLink: false,
-      },
-      mapYesNoField(answers[TOTAL_CONTRACT_VALUE_ELIGIBILITY].valueId === MORE_THAN_500K.DB_ID),
-    ),
```

Comment:
Is the "total contract value" field not required in the eligibility summary list?

Reply:
Will be adding in the next PR